project/Build.scala

```diff
@@ -12,6 +12,7 @@ object BuildSettings {
     organization := "com.plameno",
     version := "0.1-SNAPSHOT",
     scalaVersion := "2.10.2",
+    crossScalaVersions := Seq("2.11.7"),
```

Comment:
Isn't it? :) 
You can test both versions by adding the `+` sign before an SBT command.